### ISSUE TITLE
broken: ext-imageSprite cannot be added in ext-drawContainer's Items

### ISSUE THREAD

**Issue by fabriciomurta** [extnet/Ext.NET#1829]
Found: 7.0.0-rc-20200914

The `ext-imageSprite` component is not being accepted within an `ext-drawContainer`'s `items` block. The issue is as simple as:

```
<ext-drawContainer width="320" height="480">
    <items>
        <ext-imageSprite
            src="bd.jpg"
            x="0"
            y="0"
            width="320"
            height="240" />
    </items>
</ext-drawContainer>
```

This issue is likely to be the same as #1798 and #1799.

*Note:* all properties in the `<ext-imageSprite />` component are suspected not to be supported to Ext.NET 7, according to intellisense syntax highlighting:
![image](https://user-images.githubusercontent.com/10730372/93282566-0d1eac80-f7a5-11ea-87bf-f0feb1b7313b.png)

**Comment by AndreyChechel** [comment]
Instead of `<items>` collection, the `<sprites>` collection should be used. 
The missing attribute were fixed in `7.0.0`.

![image](https://user-images.githubusercontent.com/16582701/93324975-4bf14880-f81f-11ea-9f3f-06d2a6160843.png)

**Comment by fabriciomurta** [comment]
The example related to this issue is [Draw > Basic > Animation](https://examples5.ext.net/#/Draw/Basic/Animation/).
